NDTensors/test/test_combiner.jl

```diff
@@ -1,24 +1,27 @@
+@eval module $(gensym())
 using NDTensors
-using LinearAlgebra
-using Test
+using Test: @testset, @test, @test_throws
 using GPUArraysCore: @allowscalar
-## TODO headergaurd
-# include("NDTensorsTestUtils/NDTensorsTestUtils.jl")
-# using .NDTensorsTestUtils: default_rtol, devices_list
+include("NDTensorsTestUtils/NDTensorsTestUtils.jl")
+using .NDTensorsTestUtils: NDTensorsTestUtils
 
 # Testing generic block indices
 using ITensors: QN, Index
 
 @testset "CombinerTensor basic functionality" begin
-  devs = devices_list(copy(ARGS))
-  @testset "test device: $dev" for dev in devs
+  @testset "test device: $dev" for dev in NDTensorsTestUtils.devices_list(copy(ARGS)),
+    elt in [Float64, Float32]
```

Comment:
```suggestion
    elt in (Float32, Float64)
```
When putting objects with different types in a collection it is better to use a `Tuple`.

Reply:
Thanks, I updated this across all the tests files in NDTensors